packages/marshal/test/_marshal-test-data.js

```diff
@@ -49,12 +49,10 @@ export const roundTripPairs = harden([
   [9007199254740993n, { '@qclass': 'bigint', digits: '9007199254740993' }],
 
   // Well known symbols
-  [Symbol.asyncIterator, { '@qclass': 'symbol', name: '@@asyncIterator' }],
-  [Symbol.match, { '@qclass': 'symbol', name: '@@match' }],
-  // Registered symbols
-  [Symbol.for('foo'), { '@qclass': 'symbol', name: 'foo' }],
-  // Registered symbol hilbert hotel
-  [Symbol.for('@@foo'), { '@qclass': 'symbol', name: '@@@@foo' }],
+  [Symbol.asyncIterator, { '@qclass': 'symbol', name: 'Symbol.asyncIterator' }],
```

Comment:
Does this mean well known symbols are still passable?

Reply:
Good question! 

- To ease the transition, we are currently making a temporary special case for `Symbol.asyncIterator` both for endo here, and for agoric-sdk at https://github.com/Agoric/agoric-sdk/pull/11338

- Aside from that temporary special case, we treat all unregistered symbols as passable, whether well-known or not. However, when we unserialize such symbols, we create fresh ones with the same description, as we do for all unregistered symbols. Thus they lose their JS equality to the well-known symbols.